tokencost/costs.py

```diff
@@ -13,6 +13,15 @@
 # https://github.com/anthropics/anthropic-tokenizer-typescript/blob/main/index.ts
 
 
+def strip_ft_model_name(model: str) -> str:
```

Comment:
I'm confused by this one-- `ft:gpt-3.5-turbo` is the model name. Why add a specific function to check for this?

Reply:
I've also raised an issue with LiteLLM https://github.com/BerriAI/litellm/issues/1840

Reply:
TOKEN_COSTS is a dict containing prices. Every key is the name of the model. One of the key is `ft:gpt-3.5-turbo`

When using a finetuned model, the name of the model refers to the exact finetuned model you deployed in openai. The model name is ft:gpt-3.5-turbo:my-org:custom_suffix:id

At the beginning of every cost function, there is a check : is this a known model ? do we have a cost for this model ? The check is the following:
```
if model not in TOKEN_COSTS
```
This check will systematically fail if you don't strip the custom model name from `ft:gpt-3.5-turbo:my-org:custom_suffix:id` to `ft:gpt-3.5-turbo`

I added a function because `if model not in TOKEN_COSTS` is called 3 times in this script. Do not repeat yourself!

Reply:
Brilliant. I haven't used fine-tuning before, but this makes sense. Will merge.